tests/test_subclass_hierarchy.py

```diff
@@ -0,0 +1,40 @@
+import sys
+from rdflib import RDFS, Graph
+
+sys.path.append("..")
+from bricksrc.namespaces import BRICK
```

Comment:
This should have been flagged by the style checker. Can you make sure `pre-commit` is installed (see instructions @ https://github.com/BrickSchema/Brick/blob/master/CONTRIBUTING.md) and see if this gets flagged?

Reply:
You're right, I forgot to install pre-commit on my new clone.